src/commands/bootstrap/index.ts

```diff
@@ -0,0 +1,719 @@
+};
+
+type BootstrapOptions = {
+  auth?: 'auth.js';
+  framework: 'Next.js' | 'SvelteKit' | 'Nuxt.js';
+  deployment: 'vercel' | 'cloudflare';
+  orm?: 'drizzle' | 'prisma';
+
+  packageManager: 'npm' | 'pnpm' | 'bun' | 'yarn';
+};
+
+// `getCreateNextAppCommand` returns the command for creating a Next app
+// with `create-next-app` for different package managers.
+function getCreateNextAppCommand(
+  packageManager: BootstrapOptions['packageManager'],
+) {
+  const createNextAppVersion = '14.2.4';
+
+  switch (packageManager) {
+    case 'npm':
+      return `npx create-next-app@${createNextAppVersion}`;
+    case 'bun':
+      return `bunx create-next-app@${createNextAppVersion}`;
+    case 'pnpm':
+      return `pnpm create next-app@${createNextAppVersion}`;
+    case 'yarn':
+      return `yarn dlx create-next-app@${createNextAppVersion}`;
+  }
+}
+
+function getExecutorProgram(
+  packageManager: BootstrapOptions['packageManager'],
+) {
+  switch (packageManager) {
+    case 'npm':
+      return 'npx';
+    case 'pnpm':
+      return 'pnpx';
+    case 'bun':
+      return 'bunx';
+    case 'yarn':
+      return 'yarn dlx';
+  }
+}
+
+function getGlobalInstallProgram(
+  packageManager: BootstrapOptions['packageManager'],
+  packageName: string,
+) {
+  switch (packageManager) {
+    case 'npm':
+      return `npm install -g ${packageName}`;
+    case 'pnpm':
+      return `pnpm install -g ${packageName}`;
+    case 'bun':
+      return `bun add -g ${packageName}`;
+    case 'yarn':
+      return `yarn global add ${packageName}`;
+  }
+}
+
+const bootstrap = async (props: CommonProps) => {
+  const out = writer(props);
+
+  if (isCi()) {
+    throw new Error('Cannot run interactive auth in CI');
+  }
+
+  const res = await prompts({
+    onState: onPromptState,
+    type: 'text',
+    name: 'path',
+    message: 'What is your project named?',
+    initial: 'my-app',
+    validate: (name: string) => {
+      // We resolve to normalize the path name first, so that if the user enters
+      // something like "/hello", we get back just "hello" and not "/hello".
+      // This avoids issues where relative paths might lead to different results
+      // depending on the current working directory. It also prevents issues
+      // related to invalid symlinks.
+      const validation = validateNpmName(basename(resolve(name)));
+      if (validation.valid) {
+        return true;
+      }
+      return 'Invalid project name: ' + validation.problems[0];
+    },
+  });
+  trackEvent('create-app', { phase: 'start' });
+
+  if (typeof res.path !== 'string') {
+    throw new Error('Could not get project path');
+  }
+
+  // We resolve to normalize the path name first, so that if the user enters
+  // something like "/hello", we get back just "hello" and not "/hello".
+  // This avoids issues where relative paths might lead to different results
+  // depending on the current working directory. It also prevents issues
+  // related to invalid symlinks.
```

Comment:
so, when I am in a homedir and type something like `~/test-app` the app name would resolve to the "test-app"?

Reply:
and seems like `neondatabase/test-app` resolves to just "test-app"

Reply:
`~` is not allowed and yes `neondatabase/test-app` does resolve to just `test-app`